airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/json_operations.sql

```diff
@@ -50,7 +50,7 @@
 {%- endmacro %}
 
 {% macro redshift__json_extract(json_column, json_path_list) -%}
-    case when json_extract_path_text({{ json_column }}, {{ format_json_path(json_path_list) }}) != '' then json_extract_path_text({{ json_column }}, {{ format_json_path(json_path_list) }}) end
+    case when json_extract_path_text({{ json_column }}, {{ format_json_path(json_path_list) }}, true) != '' then json_extract_path_text({{ json_column }}, {{ format_json_path(json_path_list) }}, true) end
```

Comment:
The default value for the third argument of `json_extract_path_text` on redshift is `false`

which throws exceptions if invalid json are encountered. For example
```
error:  JSON parsing error
code:      8001
context:   invalid json object [{"filterFamily":"CompanyPropertyValue","operator":"SET_ANY","propertyObjectType":"COMPANY","withinTimeMode":"PAST","type":"enumeration","property":"business_type","value":"Grower"},{"filterFamily":"CompanyPropertyValue","operator":"SET
query:     72527682
location:  funcs_json.hpp:128
process:   query0_101_72527682 [pid=27692]
```

Now it would return null values instead in the unnested tables (keeping the string value in the parent table)

Reply:
nice catch!